src/elements/Button/Button.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import {
+	TouchableHighlight, FlexStyle, StyleProp, ViewStyle, TextStyle,
+} from 'react-native';
+import { ColorPalette, COLOR_SCHEMES, ColorScheme } from '@util/colorSchemes';
+import { useColorScheme } from 'react-native-appearance';
+import styles from './Button.styles';
+
+export type ButtonProps = {
```

Comment:
Add all of [TouchableHighlight properties](https://reactnative.dev/docs/touchablehighlight) (e.g. nextFocusLeft) to this button so that the component user can have more control over the button.